docker-compose.yml

```diff
@@ -17,4 +19,4 @@ services:
     volumes:
       # mount the local directory ./app-data into the container
       # (so settings, database and repository are accessible)
-      - ./app-data:/app-data
+      - otterwiki:/app-data:rw,z
```

Comment:
With this change not a local directory would be mounted, but a volume created and stored somewhere in the docker environment, e.g. `/var/lib/docker/volumes/...`, depending on the volume driver.

There is nothing to be said against this, but it must be correctly commented.

In the case that a user want to mount the local directory and use `SELINUX=enforcing` a change of the directories context via `semanage` might be the way to go.

Reply:
I was able to run podman with a mount from a local directory using

```
podman run --rm -p 8080:80 -v $PWD/app-data:/app-data:Z redimp/otterwiki:2
```

I think the documentation needs an update on running An Otter Wiki in an enviroment with `SELinux=enforcing`.

@mattock What is the current best way to get `docker compose` (or `podman compose`?) running on Rocky9?

Reply:
> With this change not a local directory would be mounted, but a volume created and stored somewhere in the docker environment, e.g. `/var/lib/docker/volumes/...`, depending on the volume driver.

Yep. The location when using podman depends on whether you're running podman/podman-compose as root or as a normal user. In the former case volume(s) end up under /var/lib/<something> like with Docker, whereas in the latter case they are placed under ~/.local/share/containers/storage/volumes/.

> There is nothing to be said against this, but it must be correctly commented.

Fully agreed.

> 
> In the case that a user want to mount the local directory and use `SELINUX=enforcing` a change of the directories context via `semanage` might be the way to go.

Semanage + restorecon + chown + chmod should work. I'm not 100% sure if just adding the "z" o "Z" volume mount option could also resolve this. In general I find using the default volume locations easier with Podman, but that's just my personal preferences.